src/main/java/com/appdynamics/extensions/oracledb/JDBCConnectionAdapter.java

```diff
@@ -21,8 +23,7 @@
 
     private final String connUrl;
     private final Map<String, String> connectionProperties;
-    private static final org.slf4j.Logger logger = LoggerFactory.getLogger(JDBCConnectionAdapter.class);
-
+    private static final Logger logger = ExtensionsLoggerFactory.getLogger(OracledbMonitor.class);
```

Comment:
Can you please fix class name for each logger. It should have class name for which logger is created. Please check this for other classes where logger is created.